### PR TITLE
Add Blip and Blip2 for pipeline tests

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

A continuation of (not merged) #21802.

@NielsRogge I will add you as the contributor.
@Narsil Just in case you want to take a look too, as you reviewed #21904, and asked for adding the tests.

### CODE REVIEW

File: utils/create_dummy_models.py [hunk 1]
```diff
@@ -410,7 +410,10 @@ def convert_processors(processors, tiny_config, output_folder, result):
         elif isinstance(processor, ProcessorMixin):
             # Currently, we only have these 2 possibilities
             tokenizers.append(processor.tokenizer)
-            feature_extractors.append(processor.feature_extractor)
+            if hasattr(processor, "image_processor"):
+                feature_extractors.append(processor.image_processor)
+            elif hasattr(processor, "feature_extractor"):
+                feature_extractors.append(processor.feature_extractor)
```
Comment (ydshieh) [inline utils/create_dummy_models.py:416]: the name of `feature_extractors` here is not good - it is a leftover when we didn't have image processor yet.

File: utils/create_dummy_models.py [hunk 2]
```diff
@@ -557,7 +560,7 @@ def upload_model(model_dir, organization):
     repo_exist = False
     error = None
     try:
-        create_repo(repo_id=repo_name, organization=organization, exist_ok=False, repo_type="model")
+        create_repo(repo_id=f"{organization}/{repo_name}", exist_ok=False, repo_type="model")
```
Comment (ydshieh) [inline utils/create_dummy_models.py:563]: API changed - I need to update it

File: utils/create_dummy_models.py [hunk 3]
```diff
@@ -778,7 +781,15 @@ def get_config_overrides(config_class, processors):
...
-    if config_class.__name__ in ["CLIPConfig", "GroupViTConfig", "OwlViTConfig", "XCLIPConfig", "FlavaConfig"]:
+    if config_class.__name__ in [
+        "CLIPConfig",
+        "GroupViTConfig",
+        "OwlViTConfig",
+        "XCLIPConfig",
+        "FlavaConfig",
+        "BlipConfig",
+        "Blip2Config",
+    ]:
```
Comment (ydshieh) [inline utils/create_dummy_models.py:792]: These models have text and vision parts. When dealing with model testers, we need to be careful regarding the `vocab_size` (i.e. pass it through `text_kwargs`)

File: utils/create_dummy_models.py [hunk 4]
```diff
@@ -410,7 +410,10 @@ def convert_processors(processors, tiny_config, output_folder, result):
         elif isinstance(processor, ProcessorMixin):
```
Comment (ydshieh) [inline utils/create_dummy_models.py:410]: A few updates are necessary to make the script work (in general, as well as for blip and blip_2)

**Review by sgugger — APPROVED** [review]
Nice addition!

**Comment by Narsil** [comment]
Thank you ! !!!

**Pull request merged**